crmd/throttle.c

```diff
@@ -102,14 +83,16 @@ int throttle_num_cores(void)
  *       This will return NULL if the daemon is being run via valgrind.
```

Comment:
Also note that with permissive options, there's another undesired
side-effect: unreleated user on the machine can do:

```
> cib cat <<EOF
#!/bin/sh
while true; do factor ${RANDOM} >/dev/null; done
EOF
chmod +x cib
./cib
```

and possibly influence the run of the whole cluster, which should not
happen, at least not so easily.

That's all due to the clunky code to get details about running CIB
daemon.  If anything else, pacemaker (mcp) knows very specific PIDs
to look at, and also exports its own API that could be perhaps used
to query the PID (or info about the process required).